proposals/templates/proposals/study_consent.html

```diff
@@ -1,15 +1,12 @@
 {% extends "base/base.html" %}
-
 {% load static %}
 {% load i18n %}
 {% load uil_filters %}
-
 {% block header_title %}
-  {% trans "Informed consent formulieren" %} - {{ block.super }}
+    {% trans "Informed consent formulieren" %} - {{ block.super }}
 {% endblock %}
-
 {% block html_head %}
-  <script>
+    <script>
```

Comment:
Could you update the branch with `format_js=true` in the config? I'm curious: might as well see what djlint is capable of

Reply:
Done, also with `format_css=true`